core/retries/src/main/java/software/amazon/awssdk/retries/DefaultRetryStrategy.java

```diff
@@ -70,9 +90,29 @@ static final class Standard {
         static final Duration MAX_BACKOFF = Duration.ofSeconds(20);
         static final int TOKEN_BUCKET_SIZE = 500;
         static final int DEFAULT_EXCEPTION_TOKEN_COST = 5;
+
+        private Standard() {
```

Comment:
Added the constructors since without these SonarCloud marks the class as a code smell.

Reply:
No, it cannot be since it's used internally for `AdaptiveRetryStrategy` to configure settings that we do not expose in the interface (e.g., `TOKEN_BUCKET_SIZE`).